source/contracts/Controlled.sol

```diff
@@ -36,7 +37,15 @@ contract Controlled is IControlled {
         return true;
     }
 
-    function suicideFunds(address _target) public onlyControllerCaller returns(bool) {
+    function suicideFunds(address _target, IUniverse _universe) public onlyControllerCaller returns(bool) {
+        // Transfer REP tokens to target
+        if(_universe) {
```

Comment:
This line will fail to compile, you need to compare `_universe` against `address(0)` to have it be a valid boolean expression.